themes/default/content/docs/support/troubleshooting.md

```diff
@@ -367,17 +367,24 @@ approach will not work, and the only way to workaround the issue is to follow th
 
 ## Recovering from an Interrupted Update {#interrupted-update-recovery}
 
-If the Pulumi CLI is interrupted when performing a deployment, you may see an error message
+If the Pulumi CLI is interrupted when performing a deployment, you may see a warning message
```

Comment:
why is this a`may` and not a `will`?

Reply:
You will only see this message if the interrupt interrupts an operation. If it is between updates (just starting, ending, or literally between updates) you won't get the warning, because no operation was interrupted.